### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by MarrisSparrisNilsson**

### CODE REVIEW

File: backend/meeting/src/express-app.js [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
+const express = require("express");
+const cors = require("cors");
+const { meeting } = require("./api");
+const { CreateChannel } = require("./utils");
+
+module.exports = async (app) => {
+	app.use(express.json());
+	app.use(
+		cors({
+			origin: ["http://localhost:3000"],
```
Comment (MarrisSparrisNilsson) [inline backend/meeting/src/express-app.js:10]: This should be changed, no?

File: backend/user/src/database/repository/user-repository.js [hunk 2]
```diff
@@ -22,27 +22,27 @@ class UserRepository {
...
 	}) {
 		const existingUser = await UserModel.findByIdAndUpdate(
-			id,
+			newId,
 			{
-				firstName,
-				lastName,
-				email,
+				newFirstName,
+				newLastName,
```
Comment (MarrisSparrisNilsson) [inline backend/user/src/database/repository/user-repository.js]: I have a fix for these names on my branch

File: backend/user/src/express-app.js [hunk 3]
```diff
@@ -1,25 +1,20 @@
...
+const express = require("express");
+const cors = require("cors");
+const { user, appEvents } = require("./api");
+const { CreateChannel, SubscribeMessage } = require("./utils");
+
+module.exports = async (app) => {
+	app.use(express.json());
+	app.use(
+		cors({
+			origin: ["http://localhost:3000"],
```
Comment (MarrisSparrisNilsson) [inline backend/user/src/express-app.js:10]: Same here?

**Review by MarrisSparrisNilsson — COMMENTED** [review]
Look at my comments. Other than that good job!

File: backend/meeting/src/express-app.js (hunk 1 above)
Comment (JoelScarinius) [reply backend/meeting/src/express-app.js:10]: It depends on the loadbalancer IP which changes evrytime we do terraform apply/destroy. But as I said I don't really think cors is a problem in production only in developement.

File: backend/user/src/database/repository/user-repository.js (hunk 2 above)
Comment (JoelScarinius) [reply backend/user/src/database/repository/user-repository.js]: I will copy you fix.

File: backend/user/src/express-app.js (hunk 3 above)
Comment (JoelScarinius) [reply backend/user/src/express-app.js:10]: It depends on the loadbalancer IP which changes evrytime we do terraform apply/destroy. But as I said I don't really think cors is a problem in production only in developement.

**Pull request merged**